deployments/prod/environment

```diff
@@ -18,4 +18,6 @@ _set AZUL_DOMAIN_NAME "explore.data.humancellatlas.org"
 
 _set AZUL_URL_REDIRECT_FULL_DOMAIN_NAME "url.data.humancellatlas.org"
 
-_set AZUL_ENABLE_CLOUDWATCH_ALARMS 1
+_set azul_grafana_endpoint "metrics.data.humancellatlas.org"
```

Comment:
The variable name ends in "endpoint" so the value needs to be a URL, not just a host name.